src/models/collaboration/useRequestCollaboration.js

```diff
@@ -25,6 +29,7 @@ export default function useRequestCollaboration() {
         setSuccess(true);
         setError(null);
         setLoading(false);
+        queryClient.invalidateQueries(queryKeys.me);
```

Comment:
I don't understand why queryKeys.me specifically was the one that had to be invalidated here.

Reply:
after creating a collaboration we refresh `api/v1/me` because that's where the collaborations are stored